images/scripts/rhel4edge.bootstrap

```diff
@@ -12,8 +12,13 @@ cat > edge-request.json <<EOF
   "imageType": "rhel-edge-installer",
   "packages": [
     { "name": "cockpit-system" },
+    { "name": "insights-client" },
+    { "name": "ncurses" },
+    { "name": "openssh-server" },
     { "name": "socat" },
-    { "name": "openssh-server" }
+    { "name": "sscg" },
+    { "name": "sssd-common" },
+    { "name": "subscription-manager-cockpit" }
```

Comment:
Why do we need to pre-install that? AFAIK we don't use it in cockpit's tests, and doesn't this make it harder to test sub-man PRs, as you would then have to replace already installed packages in the OSTree?

Reply:
It is exactly the same what we do in rhel.setup, see https://github.com/cockpit-project/bots/blob/main/images/scripts/rhel.setup#L182